src/test/headers_sync_chainwork_tests.cpp

```diff
@@ -234,4 +244,28 @@ static void TooLittleWork(const CBlockIndex* chain_start,
         /*exp_locator_hash=*/std::nullopt);
 }
 
+static void TooBigBuffer(const CBlockIndex* chain_start,
+        const std::vector<CBlockHeader>& first_chain)
+{
+    // Intentionally too big redownload buffer in order to trigger warning.
```

Comment:
nice, this is exactly how comments should be used to augment what the code cannot easily tell <3